easybuild/easyconfigs/m/Mako/Mako-1.0.7-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'Mako'
+version = '1.0.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.makotemplates.org'
+description = """A super-fast templating language that borrows the best ideas from the existing templating languages"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae']
+
+dependencies = [('Python', '3.6.4')]
+
+download_dep_fail = True
+
+use_pip = True
+
+sanity_check_paths = {
+    'files': ['bin/mako-render'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s-%(version)s-py%(pyshortver)s.egg'],
```

Comment:
@SimonPinches Sanity check fails for me here because `pip` doesn't install a `.egg` directory, just check for `'lib/python%(pyshortver)s/site-packages'` instead?

Reply:
@boegel Rather than shortening I've corrected by adding the `-info` extension.  Should work now but I can also trim if you prefer.

Reply:
I actually prefer trimming it, since that matches what we do in other recent `PythonPackage` easyconfigs: check for a non-empty `lib/python%(pyshortver)s/site-packages` directory + do an `import` test.